scripts/vagrantCloudBasedContainers.sh

```diff
@@ -51,8 +51,7 @@ docker build -t $UNIVERSAL_IMAGE .
 docker run -d -p $COUCHDB_PORT:$COUCHDB_PORT --name couchdb $COUCHDB_IMAGE
 
 # Wait for CouchDB
-sleep $COUCHDB_HEALTHCHECK_DELAY
-curl --retry $COUCHDB_HEALTHCHECK_TIMEOUT --retry-delay 1 --retry-connrefused http://localhost:$COUCHDB_PORT
+wget --retry-connrefused --waitretry=$COUCHDB_HEALTHCHECK_DELAY --read-timeout=20 --timeout=$COUCHDB_HEALTHCHECK_TIMEOUT -t 30 http://localhost:$COUCHDB_PORT
```

Comment:
COUCHDB_HEALTCHECK_TIMEOUT=30 was for the whole test to take at most 30 seconds (and fail if it didn't work within that time). I think here it's going to wait 30 seconds per retry and it'll retry 30 times (`-t 30`) so it could stay stuck for 15min?

The curl was code retrying every 1 seconds (`--retry-delay 1`) and doing 30 retries (`--retry $COUCHDB_HEALTHCHECK_TIMEOUT`)